staging/src/k8s.io/api/batch/v1beta1/types.go

```diff
@@ -105,8 +105,15 @@ type CronJobSpec struct {
 	Schedule string `json:"schedule" protobuf:"bytes,1,opt,name=schedule"`
 
 	// The time zone for the given schedule, see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones.
```

Comment:
I tweaked this to suggest `The time zone name` (since we're specifying a name, not an offset etc.  Minor clarity point.